sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -1762,6 +1772,13 @@ public partial class SearchableField : Azure.Search.Documents.Indexes.Models.Sim
         public Azure.Search.Documents.Indexes.Models.LexicalAnalyzerName? SearchAnalyzerName { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> SynonymMapNames { get { throw null; } }
     }
+    public partial class SearchAlias
+    {
+        public SearchAlias(string name, System.Collections.Generic.IEnumerable<string> indexes) { }
+        public Azure.ETag? ETag { get { throw null; } set { } }
```

Comment:
Why is this settable?

Reply:
This was to adhere to the convention I saw in other places in the Search SDK.
We expose the setter for `ETag` in `SearchIndex`, `SearchIndexer`, `SearchIndexerDataSourceConnection`, `SearchIndexerSkillset` and `SynonymMap`. I think this has been a mistake in other places as well.

Should I see which other non-GA places we can remove it from?